R/Spectra.R

```diff
@@ -365,6 +365,18 @@ NULL
 #' - `split`: splits the `Spectra` object based on parameter `f` into a `list`
 #'   of `Spectra` objects.
 #'
+#' - `joinSpectraData`: Individual spectra data variable can be
```

Comment:
I would suggest to change that sentence to: `individual spectra variables can be...`.